examples/tests/sdk_tests/util/wait.w

```diff
@@ -0,0 +1,57 @@
+bring cloud;
+bring util;
+
+class JSHelper { 
+  extern "./sleep-helper.js" static inflight getTime(): num;
+}
+let oneHundredMiliseconds = 0.1s;
+
+test "returns true immediately" {
+  let start = JSHelper.getTime();
+  if util.wait((): bool => { return true; }) {
+    assert(JSHelper.getTime() - start < 1000);
+  } else {
+    assert(false);
+  }
+}
+
+test "returns false takes a minute" {
```

Comment:
This test is taking a minute, because it is testing the default behavior of timeout (set to 1 minute)

It is a long time to wait for developers on their box, should we change the test to a quicker timeout and not verify the 1m default timeout?

Reply:
Unless someone comes up with a way around this, I think it's ok to avoid testing the default 1m. For me, it's the slight chance of ever introducing an error in this simple code vs. the huge cost × number of runs...

Reply:
fixed , thanks!